certified-connectors/Verified/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2681 @@
+              "items": {
+                "$ref": "#/definitions/Descriptor"
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "advanced"
+      }
+    },
+    "/flows/{flow-id}/jobs/{envelope-id}": {
+      "get": {
+        "operationId": "GetFlowsFlowIdJobsEnvelopeId",
+        "summary": "Get flow state by envelopeid",
+        "description": "Gets the current flow state of a given envelope",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          },
+          {
+            "name": "flow-id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Flow Id",
+            "description": "Id of the flow in question",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "token": {
+                  "type": "string",
+                  "title": "Token"
+                }
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "advanced"
+      }
+    },
+    "/envelopes/{envelope-id}/documents/{document-id}": {
+      "get": {
+        "operationId": "GetEnvelopesEnvelopeIdDocumentsDocumentId",
+        "summary": "Get document by id",
+        "description": "Gets document information by its id",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          },
+          {
+            "$ref": "#/parameters/DocumentId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/Document"
+            }
+          }
+        },
+        "x-ms-visibility": "advanced"
+      },
+      "delete": {
+        "operationId": "DeleteEnvelopesEnvelopeIdDocumentsDocumentId",
+        "summary": "Delete document by id",
+        "description": "Deletes a document from the Verified Portal.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          },
+          {
+            "$ref": "#/parameters/DocumentId"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
As stated in the API documentation this endpoint does not return any information in the body either
https://verified-dev.api-docs.io/v2.1/documents/document-delete